<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NBD Server Linux</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gui_icons.qrc">
    <normaloff>:/images/images/USB.png</normaloff>:/images/images/USB.png</iconset>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="StartB">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start NBD Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MountB">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Mount USB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UnmountB">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>UnMount USB</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Line_Account">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>root</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Line_Password">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0penBmc</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Range</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Line_IP_Start">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.127</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Line_IP_End">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.127</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopB">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop NBD Server</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_file">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BrowseB">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="gui_icons.qrc">
      <normaloff>:/images/images/browse.png</normaloff>:/images/images/browse.png</iconset>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="SDDevices">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioDevice">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Device</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchB">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BMC</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxBMC">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitB">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>le_file</tabstop>
  <tabstop>BrowseB</tabstop>
  <tabstop>SDDevices</tabstop>
  <tabstop>Line_IP_Start</tabstop>
  <tabstop>Line_IP_End</tabstop>
  <tabstop>SearchB</tabstop>
  <tabstop>comboBoxBMC</tabstop>
  <tabstop>Line_Account</tabstop>
  <tabstop>Line_Password</tabstop>
  <tabstop>StartB</tabstop>
  <tabstop>MountB</tabstop>
  <tabstop>UnmountB</tabstop>
  <tabstop>StopB</tabstop>
  <tabstop>radioDevice</tabstop>
  <tabstop>radioImage</tabstop>
 </tabstops>
 <resources>
  <include location="gui_icons.qrc"/>
 </resources>
 <connections/>
</ui>
